src/worldstate/models/items/mod.rs

```diff
@@ -0,0 +1,277 @@
+//! Everything to do with the item type - whether it's Warframes, Weapons, Mods, everything.
+
+use arcane::Arcane;
+use archwing::Archwing;
+use chrono::NaiveDate;
+use fish::Fish;
+use gear::Gear;
+use glyph::Glyph;
+use misc::Misc;
+use modification::Mod;
+use node::Node;
+use pet::Pet;
+use quest::Quest;
+use relic::Relic;
+use resource::Resource;
+use sentinel::Sentinel;
+use serde::Deserialize;
+use sigil::Sigil;
+use skin::Skin;
+use warframe::Warframe;
+use weapon::Weapon;
+
+pub mod arcane;
+pub mod archwing;
+pub mod fish;
+pub mod gear;
+pub mod glyph;
+pub mod misc;
+pub mod modification;
+pub mod node;
+pub mod pet;
+pub mod quest;
+pub mod relic;
+pub mod resource;
+pub mod sentinel;
+pub mod sigil;
+pub mod skin;
+pub mod warframe;
+pub mod weapon;
+
+/// Represents a polarity
+#[derive(Clone, Copy, Debug, Deserialize, PartialEq, Eq, strum::Display)]
+pub enum Polarity {
+    /// V (Damage, Powers) - Commonly dropped by Grineer
+    #[serde(rename = "madurai")]
+    Madurai,
+
+    /// D (Defensive, Health, Armor) - Dropped by all factions
+    #[serde(rename = "vazarin")]
+    Vazarin,
+
+    /// Dash/Bar (Utility, Misc.) - Commonly dropped by Corpus
+    #[serde(rename = "naramon")]
+    Naramon,
+
+    /// Mainly used for Warframe Augment Mods, in addition to some Melee Stance Mods
+    #[serde(rename = "zenurik")]
+    Zenurik,
+
+    /// Used for certain Melee Stance Mods
+    #[serde(rename = "unairu")]
+    Unairu,
+
+    /// Y (Companion Abilities) - Dropped by all factions
+    #[serde(rename = "penjaga")]
+    Penjaga,
+
+    /// U (Anti-Sentient Mods) - Obtained upon completion of The Sacrifice
+    #[serde(rename = "umbra")]
+    Umbra,
+
+    /// O (Universal polarity) - Can be only applied by Aura- and Stance-Forma on their slots
+    /// respectively
+    #[serde(rename = "any")]
+    Any,
+}
```

Comment:
Is there a reason we don't just use `rename_all` to be inline with Warframes etc. and the other models?
```suggestion
#[derive(Clone, Copy, Debug, Deserialize, PartialEq, Eq, strum::Display)]
#[serde(rename_all = "camelCase")]
pub enum Polarity {
    /// V (Damage, Powers) - Commonly dropped by Grineer
    Madurai,

    /// D (Defensive, Health, Armor) - Dropped by all factions
    Vazarin,

    /// Dash/Bar (Utility, Misc.) - Commonly dropped by Corpus
    Naramon,

    /// Mainly used for Warframe Augment Mods, in addition to some Melee Stance Mods
    Zenurik,

    /// Used for certain Melee Stance Mods
    Unairu,

    /// Y (Companion Abilities) - Dropped by all factions
    Penjaga,

    /// U (Anti-Sentient Mods) - Obtained upon completion of The Sacrifice
    Umbra,

    /// O (Universal polarity) - Can be only applied by Aura- and Stance-Forma on their slots
    /// respectively
    Any,
}
```

Reply:
Yea, when I made this enum I was on autopilot. Good point, alltho I think simply 'lowercase' would be better, since the deserializer wouldn't have to do the camelCase check (a tiny optimization)